TOC.md

```diff
@@ -505,7 +505,7 @@
     + [pd-ctl](/pd-control.md)
     + [tidb-ctl](/tidb-control.md)
     + [pd-recover](/pd-recover.md)
-  + Command Line Flags
+  + Command-line Parameters
```

Comment:
See the file names "command-line" flags. All file names use "flag". If the most majority of usages are "flag" while very few are "parameters" in this "command-line" series of documents, why bother changing the majority for the few? See https://docs.pingcap.com/tidb-data-migration/stable/command-line-flags.